operator/v1/types_ingress.go

```diff
@@ -1453,6 +1453,33 @@ type IngressControllerTuningOptions struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	MaxConnections int32 `json:"maxConnections,omitempty"`
+
+	// reloadInterval defines the minimum interval at which the router is allowed to reload
+	// to accept new changes. Increasing this value can prevent the accumulation of
+	// HAProxy processes, depending on the scenario. Increasing this interval can
+	// also lessen load imbalance on a backend's servers when using the roundrobin
+	// balancing algorithm. Alternatively, decreasing this value may decrease latency
+	// since updates to HAProxy's configuration can take effect more quickly.
+	//
+	// The value must be a time duration value; see <https://pkg.go.dev/time#ParseDuration>.
+	// Currently, the minimum value allowed is 1s, and the maximum allowed value is
+	// 120s. Minimum and maximum allowed values may change in future versions of OpenShift.
+	//
+	// An empty reloadInterval tells the IngressController to choose the default, which
+	// is currently 5s.
+	//
+	// This field expects an unsigned duration string of decimal numbers, each with optional
+	// fraction and a unit suffix, e.g. "100s", "1m30s". Valid time units are "s" and "m".
+	//
+	// Note: Setting a large value (e.g. > 90s) can cause significant latency in observing
```

Comment:
Also indicate that the ingress operator will reject values greater than 90s and use a default determined by the IngressController, that is subject to change in the future and is currently 5s.

Reply:
Miciah asked me to set the max as 120s. Should I change this to 90s then? The current code also caps values instead of rejects them (i.e. if a user sets a value over 120s, it gets capped to 120s and not replaced by the default). I can state this more clearly in the comments.

Reply:
Done